lib/cosmos/tools/tlm_viewer/widgets/limits_widget.rb

```diff
@@ -20,7 +20,7 @@ class LimitsWidget < Qt::Label
 
     def initialize(parent_layout, target_name, packet_name, item_name, value_type, width, height)
       super(target_name, packet_name, item_name, value_type)
-      @value_type = :CONVERTED if @value_type == :WITH_UNITS
+      raise "Invalid value_type #{@value_type} for LimitsWidget" if @value_type == :RAW
```

Comment:
Should this be if value type not equal to CONVERTED?

Reply:
No because it can be used as a part of a multiwidget like LABELTRENDLIMITSBAR which displays the value with formatted or with units. Check out the demo INST HS screen.